Documentation/configuration/metrics.rst

```diff
@@ -176,7 +176,7 @@ Endpoint
 Name                                         Labels                                             Description
 ============================================ ================================================== ========================================================
 ``endpoint_count``                                                                              Number of endpoints managed by this agent
```

Comment:
I believe this should also be changed to `endpoint`, similar to how `policy_count` was changed to `policy`?

Reply:
good pick, I missed for both :)